cwms-data-api/src/test/java/cwms/cda/api/LookupTypeControllerIT.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.api;
+
+import cwms.cda.data.dto.LookupType;
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import fixtures.TestAccounts;
+import static io.restassured.RestAssured.withArgs;
+import io.restassured.filter.log.LogDetail;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import static cwms.cda.security.KeyAccessManager.AUTH_HEADER;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@Tag("integration")
+final class LookupTypeControllerIT extends DataApiTestIT {
+
+    private static final String CATEGORY_IT = "AT_EMBANK_STRUCTURE_TYPE";
+    private static final String PREFIX_IT = "STRUCTURE_TYPE";
+    @Test
+    void test_get_create_delete() throws IOException {
+        InputStream resource = this.getClass().getResourceAsStream("/cwms/cda/api/lookup_type.json");
+        assertNotNull(resource);
+        String json = IOUtils.toString(resource, StandardCharsets.UTF_8);
+        assertNotNull(json);
+        LookupType lookupType = Formats.parseContent(new ContentType(Formats.JSON), json, LookupType.class);
+
+        // Structure of test:
+        // 1)Create the LookupType
+        // 2)Retrieve the LookupType and assert that it exists
+        // 3)Delete the LookupType
+        // 4)Retrieve the LookupType and assert that it does not exist
+        TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+
+        //Create the lookup type
+        given()
```

Comment:
The formatting for these restassured tests have specific indenting - given, when, then assertThat all in the same column and everything else a single tab indent.

Reply:
reformatted